generate/generate.go

```diff
@@ -115,6 +115,34 @@ func Load(cfg *config.Tree) ([]LoadResult, error) {
 		res.Files = generated
 		results[i] = res
 	}
+
+	for _, dircfg := range cfg.Root().Dirs() {
+		res := LoadResult{Dir: dircfg.ProjDir()}
+		evalctx, err := eval.NewContext(dircfg.Dir())
```

Comment:
The suggestion is not complete, just me thinking out loud. Feel free to ignore, just getting my head around the dirs/nodes/tree/etc abstractions.

```suggestion
	for _, node := range cfg.Root().Collect() {
		res := LoadResult{Dir: node.ProjDir()}
		evalctx, err := eval.NewContext(node.Dir())
```

Or instead of node just redefine `cfg`. Not sure it is better, just tossing the idea around.